lib/algo/DMDc.h

```diff
@@ -204,10 +238,11 @@ class DMDc
      * @param[in] dt d_dt
      * @param[in] t_offset d_t_offset
      * @param[in] state_offset d_state_offset
+     * @param[in] basis d_basis
```

Comment:
`d_basis` -> `basis`
For backwards compatibility, should this have default value `basis = nullptr`? Before this PR, the member `d_basis` was set by the class, not the user. In what cases should this be input, and why is not overwritten by the class? Some documentation on this would be helpful.

Reply:
Good point! I changed the line of code to match your suggestion and added some documentation in the hpp file (maybe it would be more helpful elsewhere?)

The idea is that for offline DMDc we compute the basis. For the interpolated DMDc we also interpolate the basis, and then we need to store it so that we can later project the controls (see line 678 in `DMDc.cpp`)